src/plone/restapi/services/configure.zcml

```diff
@@ -36,10 +36,7 @@
   <include package=".users"/>
   <include package=".vocabularies"/>
   <include package=".workflow"/>
-  <include
-      package=".workingcopy"
-      zcml:condition="have plone-5"
-      />
+  <include package=".workingcopy"/>
   <include
```

Comment:
all other ``zcml:condition="have plone-5"`` could be removed. unless this exceeds the scope of this PR, which I don't know.